tests/lib/SimpleSAML/Utils/AttributesTest.php

```diff
@@ -57,10 +58,8 @@ public function testGetExpectedAttributeNonNormalizedArray()
             'attribute' => 'value',
         ];
         $expected = 'attribute';
-        $this->expectException(
-            \InvalidArgumentException::class,
-            'The attributes array is not normalized, values should be arrays.'
-        );
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('The attributes array is not normalized, values should be arrays.');
         Attributes::getExpectedAttribute($attributes, $expected);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)